client/app/routes/routes.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Route } from 'react-router';
+import Layout from '../layout/Layout';
+import TestReactRouter from '../components/TestReactRouter';
+import TestReactRouterRedirect from '../components/TestReactRouterRedirect';
+import CommentScreen from '../components/CommentScreen';
+
+export default (
+  <Route component={Layout}>
+    <Route
```

Comment:
Whatever renders here will go into the this.props.children for the Layout component!